app/design/frontend/base/default/template/securesubmit/form.phtml

```diff
@@ -135,8 +135,10 @@ SecureSubmitMagento.init({
 //]]>
 </script>
 
+<?php if ($_loggedIn && $allow_card_saving && count($customerStoredCards)): ?>
 <script type="text/javascript">
 (".saved-card-selector input").on('selected', function () {
     (this).parent().toggleClass('selected');
  });
 </script>
+<?php endif ?>
```

Comment:
@ttnnkkrr This looks to wrap the Javascript to output only when necessary (which is good), but the Javascript still looks to be broken. Can you correct that as well and ensure it's compatible with Prototype (it appears to be jQuery-ish)?